lib/forms/pegasus_form_validation.rb

```diff
@@ -1,4 +1,5 @@
 require_relative 'pegasus_form_errors'
+require 'validates_email_format_of'
```

Comment:
Also if I get the filename wrong, then `dashboard-server` fails to start:

With `require 'i_dont_exist'`:

```
~ bin/dashboard-server

10:05:06 [rerun] Dashboard launched
10:05:09 [rerun] Watching /Users/andrew/Code/code-dot-org/lib, /Users/andrew/Code/code-dot-org/shared/middleware, /Users/andrew/Code/code-dot-org/pegasus for **/*.{rb,ru,yml} (ignoring **/migrations/*.rb) using Darwin adapter
/Users/andrew/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:293:in `require': cannot load such file -- i_dont_exist (LoadError)
```